pkgs/development/python-modules/python-openid/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, python-openid
+, django
+, nose
+, twill
+, pycrypto
+}:
+
+buildPythonPackage rec {
+  pname = "python-openid";
+  version = "2.2.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vvhxlghjan01snfdc4k7ykd80vkyjgizwgg9bncnin8rqz1ricj";
+  };
+
+  propagatedBuildInputs = [
+    django
+    twill
+    pycrypto
```

Comment:
Are these really runtime dependencies? I don't see any `install_requires` in https://github.com/openid/python-openid/blob/2.2.5/setup.py.

Reply:
But they are required by the test runner: https://github.com/openid/python-openid/blob/f7e13536f0d1828d3cef5ae7a7b55cabadff37fc/tox.ini

Reply:
Then they probably belong in `checkInputs`